File: cpp/06for/For20.cpp
#include <iostream>
using namespace std;

int main()
{
	//Task("For20");
	int N;
	cin >> N;
	double fact = 1, sum = 0;
	for (int i = 1; i <= N; i++)
	{
		fact *= i;
		sum += fact;
	}
	cout << sum;
	
	return 0;
}

File: php/06for/for20.php
<?php
$fact = 1;
$sum = 0;
$N = mt_rand(1, 50);
echo "Ҳудуди давр: ".$N."<br>";
for($i = 1; $i <= $N; $i++){
	$fact *= $i;
	$sum += $fact;
}
echo "Cуммаи факториалҳо = ".$sum;
?>